includes/controllers/class-user.php

```diff
@@ -752,6 +760,66 @@ public function delete_user( $request ) {
 		return hp\rest_response( 204 );
 	}
 
+	/**
+	 * Resend user verification email.
+	 *
+	 * @param WP_REST_Request $request API request.
+	 * @return WP_Rest_Response
+	 */
+	public function resend_email_verify( $request ) {
+
+		// Check authentication.
+		if ( is_user_logged_in() ) {
+			return hp\rest_error( 403 );
+		}
+
+		// Check permissions.
+		if ( ! get_option( 'hp_user_verify_email' ) ) {
+			return hp\rest_error( 403 );
+		}
+
+		// Get user email.
+		$email = sanitize_email( $request->get_param( 'email' ) );
+
+		if ( ! $email ) {
+			return hp\rest_error( 403, esc_html__( 'Email is not found', 'hivepress' ) );
```

Comment:
It's ok to just throw 400 error (invalid request), avoid extra translatable strings where possible. In this case it's ok because the email will be present for sure unless users try to tamper with the URL parameters.